default/profiles/front50-local.yml

```diff
@@ -1,10 +1,6 @@
 spinnaker:
   s3:
     versioning: false
-policy:
```

Comment:
@girichinna27  Why we are deleting this?? 
It wont impact anywhere ??

Reply:
That's not required in front50-local.yaml file;  Already we have the same configuration into orca-local.yaml file. If we have in 2 files - it will make 2 policy calls; out of these 2, one will be duplicate call. So, Sudhakar suggested me this change [Ref jira: OP-20405]